### PR TITLE
[SMALLFIX] Fix read timeout bug

### PR DESCRIPTION
**Pull request by peisun1115**

### CODE REVIEW

File: tests/src/test/java/alluxio/client/FileInStreamIntegrationTest.java [hunk 1]
```diff
@@ -370,4 +371,32 @@ public void run() {
...
+        os.write(bytes);
+      }
+    }
+
+    OpenFileOptions options = OpenFileOptions.defaults().setReadType(ReadType.CACHE_PROMOTE);
+    try (FileInStream in = mFileSystem.openFile(filePath, options)) {
+      byte[] buf = new byte[8 * Constants.MB];
+      while (in.read(buf) != -1) {
+      }
+    }
```
Comment (yupeng9) [inline tests/src/test/java/alluxio/client/FileInStreamIntegrationTest.java]: There is verification of the test success?

Comment (peisun1115) [inline tests/src/test/java/alluxio/client/FileInStreamIntegrationTest.java]: it timeout without the change.

Comment (yupeng9) [inline tests/src/test/java/alluxio/client/FileInStreamIntegrationTest.java]: I see. Can you comment on it, and put a timeout annotation?

Comment (peisun1115) [inline tests/src/test/java/alluxio/client/FileInStreamIntegrationTest.java]: done

**Comment by yupeng9** [comment]
Thanks for the bug fix!

**Pull request merged**